src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorDialog.java

```diff
@@ -14,39 +14,50 @@
 
 public class VisualEditorDialog extends WikiBasePageObject {
 
-  @FindBy(css = ".oo-ui-window-ready .oo-ui-frame")
-  private WebElement iframe;
+  @FindBy(css = ".oo-ui-window-ready .oo-ui-window-frame")
+  private WebElement frame;
   @FindBy(css = ".oo-ui-window-ready")
-  private WebElement dialog;
-  @FindBy(css = ".oo-ui-icon-close")
+  protected WebElement dialog;
+  @FindBy(css = ".oo-ui-window-ready .oo-ui-icon-close")
   private WebElement closeButton;
 
   public VisualEditorDialog(WebDriver driver) {
     super(driver);
+    waitForDialogVisible();
   }
 
+  @Deprecated
   public void switchToIFrame() {
     waitForElementByElement(dialog);
     waitForElementVisibleByElement(dialog);
```

Comment:
I am leaving the method here to remind myself to remove the calling of this method as I work further into other dialogs.